modules/selenium/src/main/java/org/testcontainers/containers/BrowserWebDriverContainer.java

```diff
@@ -122,14 +127,16 @@ public BrowserWebDriverContainer(DockerImageName dockerImageName) {
         recordingMode = VncRecordingMode.SKIP;
     }
 
+    @Deprecated
```

Comment:
Would be much appreciated here to have a reasoning of deprecation along with the possible options to migrate

Reply:
That's a good point - this should have a Javadoc comment like the method below.

Reply:
Indeed, I used the description of `withDesiredCapabilities()`, since the same reason applies.

Reply:
yep, thank you